documentation/source/newsfragments/1403.feature.rst

```diff
@@ -0,0 +1,15 @@
+Cocotb now supports force/release/freeze actions on simulator handles, exposing functionality similar to the respective verilog assignments to cocotb.
```

Comment:
```suggestion
Cocotb now supports deposit/force/release/freeze actions on simulator handles, exposing functionality similar to the respective Verilog/VHDL assignments.
```
VHDL-2008 also supports (part of) this: https://www.doulos.com/knowhow/vhdl_designers_guide/vhdl_2008/vhdl_200x_small/#force

Reply:
Sounds better indeed, applied!